rustls/src/conn.rs

```diff
@@ -189,6 +189,20 @@ mod connection {
                 (false, false) => Err(io::ErrorKind::WouldBlock.into()),
             }
         }
+
+        /// Obtain a chunk of plaintext data received from the peer over this TLS connection.
+        /// This does the same thing as [`Reader::fill_buf`], but unlike that method, this
+        /// method returns a slice whose lifetime is bounded by the [`ConnectionCommon`]
+        /// that created this `Reader`.
+        pub fn into_first_chunk(self) -> io::Result<&'a [u8]> {
```

Comment:
Since this is not covered by the commit description ("Implement BufRead for Reader"), please split it into a separate commit.